dev-util/clion/clion-2024.3.3.ebuild

```
@@ -75,18 +75,27 @@ src_prepare() {
 
 	rm -rv "${remove_me[@]}" || die
 
+	# removing debug symbols and relocating debug files as per #876295
+	# we're escaping all the files that contain $() in their name
+	# as they should not be executed
+	find . -type f ! -regex '.*\$\([^)]+\).*' -exec sh -c '
```

```suggestion
	find . -type f ! -name '*$(*)*'
```
I think this is simpler and looks more readable